calculator/src/pages/CalculatorPage.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import Calculator from '../components/Calculator';
```

Comment:
- [ ] Please I am having this error from my end, I think it is because the calculator component is wrongly imported, use `lowercase c` for the calculator component and you app should run normally. Something like so: 
```
import Calculator from '../components/calculator';
```
- [ ] Please also ensure that your components directory, inside the `calculator.js` file, your css file is correctly imported, kindly use a `lowercase c in calculator.css`. Something like so: 
```
import './calculator.css';
```
Checkout the screenshot below and kindly fix the issue.
![importCalculator](https://user-images.githubusercontent.com/65852621/180355047-b8ac88d1-427f-4233-b76f-7bd3b1f6ed43.png)

Reply:
Hi @Mihndim2020, thank you for the review but this works very well on my end with no errors